manual/vcredist2008/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,15 @@
+﻿$packageName = 'vcredist2005'
+$packageSearch = "Microsoft Visual C++ 2008 Redistributable*"
```

Comment:
That was from original package and I didn't change uninstaller. Will fix it now.